doc/nrf/libraries/networking/nrf_cloud.rst

```diff
@@ -133,12 +133,12 @@ The device passes this information by writing a ``fota_v2`` field containing an
 :c:func:`nrf_cloud_service_info_json_encode` can be used to generate the proper JSON data to enable FOTA.
 Additionally, :c:func:`nrf_cloud_shadow_device_status_update` can be used to generate the JSON data and perform the shadow update.
 
-Following are the three supported FOTA types:
+Following are the supported FOTA types:
 
-* ``APP``
-* ``MODEM``
-* ``BOOT``
-* ``MDM_FULL``
+* ``"APP"`` - updates the application.
+* ``"BOOT"`` - updates the `upgradable second-stage bootloader`_.
```

Comment:
```suggestion
* ``"BOOT"`` - updates the :ref:`upgradable bootloader`.
```
We have a tag in the target document page, you can use it and dump the entry in the links.txt file.
(Sorry for not spotting this earlier.)

Reply:
This is not working:

```
Warning, treated as error:
/home/runner/work/sdk-nrf/sdk-nrf/ncs/nrf/doc/_build/nrf/src/libraries/networking/nrf_cloud.rst:139:undefined label: 'upgradable bootloader'
```

Reply:
It needs an underscore between the words -> upgradable_bootloader.